src/render.rs

```diff
@@ -534,11 +534,11 @@ mod tests {
     #[test]
     fn image_alignment() {
         let text =
-            "<img src=\"https://img.shields.io/crates/v/clap.svg\" alt=\"\" align=\"center\">\n";
+            "<p align=\"center\"><img src=\"https://img.shields.io/crates/v/clap.svg\" alt=\"\"></p>\n";
         let result = markdown_to_html(text, None);
         assert_eq!(
             result,
-            "<img src=\"https://img.shields.io/crates/v/clap.svg\" alt=\"\" align=\"center\">\n"
+            "<p align=\"center\"><img src=\"https://img.shields.io/crates/v/clap.svg\" alt=\"\"></p>\n"
```

Comment:
aren't these cases already sufficiently tested by the `text_alignment` test above?

Reply:
The `text_alignment` test guarantees that the `align` attribute can be set for `<h1>` and `<h5>` HTML tags. By being explicit in this other test, we make sure that centering images will always be supported by catching any possible future regression. Testing is cheap, finding regressions is hard.

Reply:
shouldn't it be renamed to `paragraph_alignment` then? I guess the `<p>` doesn't care whether a `<img>` is inside or not 😅

Reply:
You are right that the `<p>` does not care currently. But maybe in a future release the `<p>` will care because of implementation changes. The usecase I want to cover is centering images in `README.md` on crates.io, so I make sure that this particular usecase is tested and will continue to work in the future.

It also as a side effect documents how to center images on crates.io for users, which can be searched for with the keywords "center" and "image" in the source code of the project.